src/promptflow-devkit/promptflow/_cli/_pf/_service.py

```diff
@@ -284,10 +308,11 @@ def show_service():
         log_file = HOME_PROMPT_FLOW_DIR / PF_SERVICE_LOG_FILE
     else:
         log_file = get_current_env_pfs_file(PF_SERVICE_LOG_FILE)
+    extra_info = {"log_file": log_file.as_posix(), "local_promptflow_version": get_pfs_version()}
```

Comment:
why not?
local_promptflow_version -> version?

Reply:
since it's not pfs version(I think it's better to call heartbeat to get pfs version)